homeassistant/components/vallox/strings.json

```diff
@@ -0,0 +1,19 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Vallox",
+        "description": "Set up the Vallox integration. If you have problems with configuration go to https://www.home-assistant.io/integrations/vallox.",
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "name": "[%key:common::config_flow::data::name%]"
+        }
+      }
+    },
+    "error": {
+      "already_configured": "Device is already configured",
```

Comment:
This should be an `abort` instead of an `error`.  There is also an existing common key for it.

Reply:
Should be fixed now (in https://github.com/slovdahl/core/commit/cad3f96858782a6d0d016fcb97d7a02c0434a4a0).